_component_design/chips.md

```diff
@@ -7,38 +7,93 @@ usage: >
   Chips represent a conceptual unit, such as a user, a device, a filter term, or a chunk of metadata and provide the user with the visual understanding of this conceptual unit as well as the ability to interact with them.
 preview-image: preview-images/chip.svg
 resource: true
-status: in-progress
-last-modified: 2017-08-17
+status: stable
+last-modified: 2017-11-16
 ---
 
 {% include toc.html %}
 
+## Introduction
+
 <div class="hxRow">
```

Comment:
Ah, sorry. With the landing of #328, you'll need to add the `markdown="1"` attribute to the row divs. Forgot about that one :)

Reply:
added.